src/main/java/app/gpx_animator/ui/cli/CommandLineConfigurationFactory.java

```diff
@@ -180,9 +188,13 @@ public CommandLineConfigurationFactory(final String... args) throws UserExceptio
                         case ZOOM -> cfg.zoom(Integer.parseInt(args[++i]));
                         case VERSION -> {
                             try (var pw = new PrintWriter(new OutputStreamWriter(System.out, StandardCharsets.UTF_8))) {
-                                pw.println(Constants.APPNAME_VERSION);
+                                pw.print(Constants.APPNAME_VERSION);
+                                pw.print(" (");
```

Comment:
I would prefer the output on a new line.

Reply:
Sure, no problem!